patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -1331,6 +1331,18 @@
 +        if (delegate.getResourceName() != null) return delegate.getResourceName().toString();
 +        return registryName != null ? registryName.toString() : null;
 +    }
++
++    /**
++     * Add information to the blocks tooltip, called from the default implementation of {@link ItemBlock#addInformation(ItemStack, EntityPlayer, List, boolean)}
++     * @param stack The stack the tooltip is being retrieved for
++     * @param player The player retrieving the tooltip
++     * @param tooltip The current tooltip
++     * @param advanced
```

Comment:
I traced the call hierarchy for `addInformation` all I could find was that that parameter was `GameSettings#advancedItemTooltips` which doesn't tell me anything useful to put in the doc.